schemas/v1/13_vm_messages_index.go

```diff
@@ -5,8 +5,9 @@ func init() {
 		13,
 		`
 ALTER TABLE {{ .SchemaName | default "public"}}.vm_messages
-    ADD COLUMN "index" BIGINT NOT NULL,
-    ADD CONSTRAINT vm_messages_uniq_index UNIQUE (height, state_root, cid, source, index);
+    ADD COLUMN "index" BIGINT NOT NULL DEFAULT -1;
+
+ALTER TABLE {{ .SchemaName | default "public"}}.vm_messages ALTER COLUMN "index" DROP DEFAULT;
```

Comment:
Maybe add a comment in the migration that here you are dropping the default value so that it can be dropped right away for existing rows.

Reply:
will do.